doc/user/zebra.rst

```diff
@@ -520,6 +520,39 @@ entered within the VRF context the static routes are created in the VRF.
    ip route 10.0.0.0 255.255.255.0 10.0.0.2 vrf r1-cust1 table 43
    show ip table vrf r1-cust1 table 43
 
+By using the :option:`-n` option, the *Linux network namespace* will be mapped
+over the *Zebra* VRF. One nice feature that is possible by handling *Linux
+network namespace* is the ability to name default VRF. At startup, *Zebra*
+discovers the available *Linux network namespace* by parsing folder
+`/var/run/netns`. Each file stands for a *Linux network namespace*, but not all
+*Linux network namespaces* are available under that folder. This is the case for
+default VRF. It is possible to name the default VRF, by creating a file, by
+executing following commands.
+
+.. code-block:: shell
+
+   touch /var/run/netns/vrf0
+   mount --bind /proc/self/ns/net /var/run/netns/vrf0
+
+Above command illustrates what happens when the default VRF is visible under
+`var/run/netns/`. Here, the default VRF file is `vrf0`.
+At startup, FRR detects the presence of that file. It detects that the file
+statistics information matches the same file statistics information as
+`/proc/self/ns/net` ( through stat() function). As statistics information
+matches, then `vrf0` stands for the new default namespace name.
+Consequently, the VRF naming `Default` will be overriden by the new discovered
+namespace name `vrf0`.
+
+For those who don't use VRF backend with *Linux network namespace*, it is
+possible to statically configure and recompile FRR. By using `DFLT_VRF_NAME`
```

Comment:
What happens if there is linux VRF name "global" (DFLT_VRF_NAME) should the code use the linux vrf or just continue to use the core/default linux instance?

Reply:
the doc need some update. thanks for pointing out this.

Reply:
Lou, I just changed the documentation.
if you can have a look, please.
tks

Reply:
the DFLT_VRF_NAME is now in config.h file.
by default it is set to Default.
if confiugre option is used, this will take the default value.